FAQ.md

```diff
@@ -51,10 +51,25 @@ Please see the [TOC Principles](https://github.com/cncf/toc/blob/main/PRINCIPLES
 
 The CNCF provides a variety of services that are accessible by maintainers via the ServiceDesk: https://github.com/cncf/servicedesk
 
-## What is the definition of an end user?
-End user companies use [cloud native](https://github.com/cncf/toc/blob/main/DEFINITION.md) technologies internally, but do not sell any cloud native services externally. Examples of end user companies are Adidas, Apple, Capital One, Spotify, and The New York Times. Examples of cloud native vendors (and so not end users) include cloud providers (e.g., Alibaba Cloud, AWS, Google Cloud, Microsoft Azure), consulting firms (e.g., Accenture, Booz Allen), infrastructure software vendors (e.g., SUSE, Red Hat), and telecom operators (e.g., AT&T, China Mobile).
+## What is the definition of an adopter?
+Adopters of a CNCF project are any organization that successfully leverages that project in the manner it was intended or repackages it as a core component of a service offering. To increase understanding of how the project is adopted, we distinguish a direct adopter from a transitive adopter; i.e. a Service Provider using Kubernetes as the underlying container orchestrator for their Knative offering, in this scenario the Service Provider is a direct adopter of Kubernetes and their Knative users are direct adopters of Knative but transitive adopters of Kubernetes.
```

Comment:
```suggestion
## What are direct and transitive Adopters?
We distinguish between _direct_ and _transitive_ adopters. 
_Direct_ Adopters of a CNCF project are any organization which successfully uses said project in the manner it was intended (End User) and/or provides it as part of a service (Service Provider). 
_Transitive_ adopters consume projects, commonly from Service Providers.

As an example, an organization using Kubernetes as the underlying container orchestrator for their Knative offering would be a _direct_ Adopter of both Kubernetes and Knative. Their customers would be _direct_ Adopters of Knative and _transitive_ adopters of Kubernetes.
```

Reply:
Wait, I was with you until the example.  How are those customers direct adopters of Knative.  Shouldn't they be transitive adopters of both?

Reply:
Also ... what is the language "in the manner in which it was intended" meant to convey?  For example, if a bus company use Kubernetes to update mileage software on their vehicle fleet, do they not count as end-users because they hacked it too much?  What are we excluding here?

Reply:
Provided additional clarity in line 57, please take a look

Reply:
I'm still not understanding the example; why is the non-SP org a direct adopter of Knative in this example?   (given that I'm likely to directly help with the eventual Knative Graduated application, this isn't an idle question, I'm asking because it will matter).

For "in the manner in which it was intended" I'm guessing that's there because we want to exclude anyone vendoring individual libraries from the definition of "adopter", as in someone who copied the GH interaction code of Prow, but nothing else, would not be an "adopter"?